config/stored_requests.go

```diff
@@ -29,6 +30,7 @@ func (dataType DataType) Section() string {
 		VideoDataType:      "stored_video_req",
 		AMPRequestDataType: "stored_amp_req",
 		AccountDataType:    "accounts",
+		ResponseDataType:   "stored_responses",
```

Comment:
Nitpick: Should this be the singular `stored_response`? What doe PBS-Java use for their metrics?

Reply:
Discussed offline. Decided not to modify due to other metrics names fomat